python2/test_typing.py

```diff
@@ -603,13 +603,54 @@ def test_orig_bases(self):
         class C(typing.Dict[str, T]): pass
         self.assertEqual(C.__orig_bases__, (typing.Dict[str, T],))
 
+    def test_naive_runtime_checks(self):
+        def naive_dict_check(obj, tp):
+            # Check if a dictionary conforms to Dict type
+            if len(tp.__parameters__) > 0:
+                return NotImplemented
```

Comment:
NotImplemented is not a great value to return here -- it's really only supposed to be used by binary operators (e.g. `__add__`) to indicate that the reverse variant should be tried (i.e. `__radd__`). Maybe raising NotImplementedError would be better?

While this is just a test, it will be used as an example and copied, and raising makes it clearer that that's an unimplemented feature of the naive check.